src/config/socket.ts

```diff
@@ -3,7 +3,7 @@ import { getHost } from './host';
 export const URL =
   process.env.NODE_ENV !== 'development'
     ? `wss://${getHost()}/websocket`
-    : `ws://127.0.0.1:5005/websocket`;
+    : `ws://${getHost()}/websocket`;

```

Comment:
@tobi-bams Did you test if `localhost:5005 `works with the WebSocket connection?

Reply:
Yes, I tested and it works